packages/settings-library/src/settings_library/rabbit.py

```diff
@@ -14,7 +14,7 @@ class RabbitDsn(AnyUrl):
 class RabbitSettings(BaseCustomSettings):
     # host
     RABBIT_HOST: str
-    RABBIT_PORT: PortInt = 5672
+    RABBIT_PORT: PortInt = PortInt(5672)
```

Comment:
MINOR: rather `parse_obj_as(PortInt, 5672)` ... otherwise it will not run validation

Reply:
well 5672 is ok no? but you are of course right.